src/graph/DeleteVerticesExecutor.cpp

```diff
@@ -0,0 +1,142 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "graph/DeleteVerticesExecutor.h"
+#include "storage/client/StorageClient.h"
+
+namespace nebula {
+namespace graph {
+
+DeleteVerticesExecutor::DeleteVerticesExecutor(Sentence *sentence,
+                                               ExecutionContext *ectx) : Executor(ectx) {
+    sentence_ = static_cast<DeleteVerticesSentence*>(sentence);
+}
+
+Status DeleteVerticesExecutor::prepare() {
+    space_ = ectx()->rctx()->session()->space();
+    expCtx_ = std::make_unique<ExpressionContext>();
+    expCtx_->setSpace(space_);
+    expCtx_->setStorageClient(ectx()->getStorageClient());
+
+    auto vidList = sentence_->vidList();
```

Comment:
Get vids in prepare(), do you want to support it get from input_ref. because most of the time through query sentence to get the vid then do delete.
if support input_ref, please move this into execute()

Reply:
Good Idea I will support it in another PR thanks :)